nipype/interfaces/base/traits_extension.py

```diff
@@ -54,7 +54,7 @@ class Str(Unicode):
 traits.DictStrStr = DictStrStr
 
 
-class BaseFile(BaseUnicode):
+class File(BaseUnicode):
```

Comment:
I'm unclear on the original justification for this separation, but dropping `BaseFile` seems fine to me.